scripts/test-upgrade.sh

```diff
@@ -22,30 +22,54 @@ BASE_DIR=$( cd "$( dirname ${0} )" && cd ../ && pwd )
 OA_DIR="$BASE_DIR/openstack-ansible"
 RPCD_DIR="$BASE_DIR/rpcd"
 
+# Create a backup of /etc/openstack_deploy, if one doesn't already exist
+# NOTE: This gets done by openstack-ansible's scripts/run-upgrade.sh script,
+#       but if we wait until then it'll be too late.
+if [[ ! -d "/etc/openstack_deploy.MITAKA" ]]; then
+  cp -a /etc/openstack_deploy /etc/openstack_deploy.MITAKA
```

Comment:
> That's how we do in OSA.

I'm fairly certain that RPCO has done it differently in the past though. It also makes more sense to me to label the directory with the release for which it works.

@d34dh0r53 and @Apsu you two seemed to be confused by this too on Thursday and Friday. Thoughts?